include/fast_float/float_common.h

```diff
@@ -220,15 +220,13 @@ struct value128 {
 /* Helper C++11 constexpr generic implementation of leading_zeroes */
 fastfloat_really_inline constexpr
```

Comment:
`constexpr` needs to be changed to `FASTFLOAT_CONSTEXPR14` here.

The comma abuse was to make this function `constexpr` from C++11 (it requires constexpr function bodies to only consist of a single return statement). I think it's OK to revert this, I don't have high ambitions to port constexpr functionality all the way back to C++11.